ActivatePayment/handler.ts

```diff
@@ -55,26 +60,61 @@ const activatePaymentTask = (
       toErrorPagopaProxyResponse(errorResponse, logger, paymentRequest.rptId)
   );
 
+function getPaymentHandlerTask(
+  pagoPaClient: IApiClient,
+  context: Context,
+  paymentRequest: PaymentActivationsPostRequest,
+  config: IConfig
+): Task<IResponseSuccessJson<PaymentActivationsPostResponse> | ErrorResponses> {
+  const TEST_RPTID =
```

Comment:
maybe you can use `RptIdFromString.encode(rptId)`, in this way you can configure it as only one string (rptId)